client_internals_test_utils.go

```diff
@@ -0,0 +1,46 @@
+//go:build hazelcastinternal && hazelcastinternaltest
+// +build hazelcastinternal,hazelcastinternaltest
```

Comment:
As far as I can research, this line means `hazelcastinternal || hazelcastinternaltest` where the first line implies `hazelcastinternal && hazelcastinternaltest`

Can you also look it up? If so, may be we can find a way to resolve this conflict

Reply:
No, comma means AND, and space means OR: https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool